docs/index.md

```diff
@@ -135,7 +135,7 @@ Sometimes, user reviews describe a plugin better than the author can:
 | Free                     | :white_check_mark: |        :white_check_mark:         |         :white_check_mark:         | :white_check_mark: |     :white_check_mark:     | :white_check_mark: |         :x:  (20$)         |       :x:  (19.99$)        |
 | OpenSource               | :white_check_mark: |        :white_check_mark:         |         :white_check_mark:         | :white_check_mark: |     :white_check_mark:     | :white_check_mark: |            :x:             |            :x:             |
 | API                      | :white_check_mark: |        :white_check_mark:         |         :white_check_mark:         | :white_check_mark: |     :white_check_mark:     |        :x:         | :material-check-bold: [^1] | :material-check-bold: [^1] |
-| Version Support          |    1.17 - 1.19     |            1.8 - 1.19             |             1.8 - 1.19             |    1.17 - 1.19     |         1.8 - 1.19         |   1.13 - 1.17.1    |         1.7 - 1.19         |         1.7 - 1.18         |
```

Comment:
If we update your version range, we should also take a quick look at the other plugins and check them if they are still up to date

Reply:
I updated the version ranges of all plugins.